.gitmodules

```diff
@@ -58,6 +58,9 @@
 [submodule "externals/mct"]
 	path = externals/mct
 	url = git@github.com:MCSclimate/MCT.git
+[submodule "components/mpas-albany-landice/src/SeaLevelModel"]
```

Comment:
Was this merge meant to bring in the YAKL update? If so, I don't see the YAKL submodule being updated here.